### ISSUE TITLE
Add one-line packet forwarded summary for to cilium monitor

### ISSUE THREAD

**Issue by tgraf** [cilium/cilium#901]
Add a mode to the BPF progs and to `cilium monitor` which prints a one line summary of packets being forwarded. Similar to `tcpdump` but with all of the high level container metadata. DEBUG output can very verbose.

**Comment by ianvernon** [comment]
@tgraf what constitutes "high level container metadata"? I need some clarity / context re: this; let's sync up next week.

**Comment by tgraf** [comment]
> @tgraf what constitutes "high level container metadata"? I need some clarity / context re: this; let's sync up next week.

What I have in mind is intelligent context:

- The endpoint was derived from a k8s pod:

```
[k8s:foo-3424234]:23234 (id 237) > [k8s:bar-2342343]:80 (id 238) 1500 bytes fowarded
[k8s:foo-3424234]:234234 (id 237) > 20.3.3.3:80 (id 238) 1500 bytes dropped (policy denied L4)
```

- The endpoint was derived from docker only:

```
[docker:client]:23234 (id 237) > [docker:server]:80 (id 238) 1500 bytes fowarded
```
We can then have flags which add additional context:
 - `-v` verbose
 - `-vv` more verbose
 - `-vnode` print node IP of source and destination node
 - `--payload LEN` print LEN bytes of packet payload

**Comment by ianvernon** [comment]
For the examples above, does the ID refer to a security ID or the ID of the endpoint?

**Comment by tgraf** [comment]
> For the examples above, does the ID refer to a security ID or the ID of the endpoint?

Good point, I think we should start calling it `identity 10` and `endpoint 200` to make it clear.

**Comment by tgraf** [comment]
Fixed in #1112
